src/sched.rs

```diff
@@ -215,6 +215,112 @@ mod sched_linux_like {
 
         Errno::result(res).map(drop)
     }
+
+    libc_bitflags! {
+        pub struct SchedFlags: c_int {
+            #[cfg(target_os = "android")]
+            SCHED_NORMAL;
+            #[cfg(target_os = "linux")]
+            SCHED_OTHER;
+            SCHED_FIFO;
+            SCHED_RR;
+            SCHED_BATCH;
+            SCHED_IDLE;
+            #[cfg(target_os = "android")]
+            SCHED_DEADLINE;
+            #[cfg(target_os = "linux")]
+            SCHED_RESET_ON_FORK;
+        }
+    }
+
+    #[repr(C)]
```

Comment:
The distinction is subtle, but this should be `repr(transparent)` instead.  See https://doc.rust-lang.org/1.26.2/unstable-book/language-features/repr-transparent.html .

Reply:
Should `nix::sched::CpuSet` also be updated to use `repr(transparent)`? I based the implementation for `SchedParam` off of it.

If so, I'm happy to open a separate PR making that change.

Reply:
Yeah, pretty much every Newtype should use `repr(transparent)`.  It wasn't available in older compilers, which is why some parts of Nix don't use it.

Reply:
Is this something you'd accept a PR for? I.e. find-and-replacing `repr(c)` with `repr(transparent)`?